### ISSUE TITLE
Olsen theme category archive pagination missing 1st item on 2nd page.

### ISSUE THREAD

**Issue by inaikem** [Automattic/themes#3060]
The Olsen theme and their category archive is exhibiting some very strange behaviour. See: p2EDhh-1ef-p2

TLDR: It looks like theme pagination is having some issues. The first post on “page 2” will not appear when the next page of posts is loaded in.

**Example:** 

- Visit a site using Olsen with Infinite Scrolling disabled.
- Visit an archive page with at least one additional page of posts.
- Click on the 'Older Posts' button.
- The second page of posts will load but is missing the first post that should be on that page.

So, for a site with 20 posts and `Manage > Settings > Content types` set to '10', we see the first 10 posts on an archive page as expected. Click the 'Older Posts' button, and only posts 12-20 are loaded in. Post 11 is skipped.

Deleting other posts has no effect as the first post on the second page is always missing.

We initially assumed this was an issue with the previous/next post content and scrubbed them clean of unnecessary HTML. This did not work and the above hypothesis was arrived at in: p1611028770046200-slack-wpcom-support

* User report: [#3656217-zd]

**Comment by scruffian** [comment]
Moved to Trac. Closing.

**Comment by Robertght** [comment]
I can see another report came in and I'm wondering where is this being tracked at the moment as I can't find it.

Also, is it ok to keep this open and report the incoming cases here?  cc @scruffian 

#4257632-zen

**Comment by Robertght** [comment]
Ignore. I meant to comment on another issue.
